pkgs/applications/terminal-emulators/cool-retro-term/default.nix

```diff
@@ -1,22 +1,22 @@
 { lib, stdenv, fetchFromGitHub, mkDerivation, qtbase, qtquick1, qmltermwidget
-, qtquickcontrols, qtgraphicaleffects, qmake, nixosTests }:
+, qtquickcontrols2, qtgraphicaleffects, qmake, nixosTests }:
 
 mkDerivation rec {
-  version = "1.1.1";
+  version = "1.2.0";
   pname = "cool-retro-term";
 
   src = fetchFromGitHub {
     owner = "Swordfish90";
     repo = "cool-retro-term";
     rev = version;
-    sha256 = "0mird4k88ml6y61hky2jynrjmnxl849fvhsr5jfdlnv0i7r5vwi5";
+    sha256 = "PewHLVmo+RTBHIQ/y2FBkgXsIvujYd7u56JdFC10B4c=";
   };
 
   patchPhase = ''
     sed -i -e '/qmltermwidget/d' cool-retro-term.pro
   '';
 
-  buildInputs = [ qtbase qtquick1 qmltermwidget qtquickcontrols qtgraphicaleffects ];
+  buildInputs = [ qtbase qtquick1 qmltermwidget qtquickcontrols2 qtgraphicaleffects ];
```

Comment:
This still needs `qtquickcontrols` on Darwin (in addition to `qtquickcontrols2`) or I get the error `qrc:/SettingsWindow.qml:25:1: module "QtQuick.Dialogs" is not installed`.

Not sure if this is a defect in Nixpkgs or Qt.

Reply:
Added `qtquickcontrols` in as a conditional dependency for Darwin.